### PR TITLE
Use compose watch

### PR DESCRIPTION
**Pull request by foodelevator**
A bit nicer and better way to develop locally using docker compose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,13 +57,18 @@ These must be set when building, not when running the application. To do this wh
 
 ### The easy way
 
+Make sure you have either podman or docker with podman-compose or docker-compose
+isntalled. You need at least version 2.22 of docker-compose since that's when
```
Comment (Herkarl) [inline README.md:61]:
```suggestion
installed. You need at least version 2.22 of docker-compose since that's when
```

**Review by Herkarl — COMMENTED** [review]
found a typo

**Comment by mathiasmagnusson** [comment]
Have you by the way tried if this works for you @Herkarl? (Including changing a file and seeing if it updates).

I just remembered that I have to run both `compose up` and `compose watch` in different terminals but that might just be a podman thing ^^

**Comment by Herkarl** [comment]
Was at the wrong computer before :p, but tried it now.
It seems to work for me! And it was enough with just `docker compose watch`, no `up` needed.

Strangely enough though the containers were not available at `localhost` for me, just `0.0.0.0`. Probably something simple, but too tired too look into it now.

**Review by Herkarl — APPROVED** [review]

**Pull request merged**